frontend/public/module/k8s/index.ts

```diff
@@ -857,6 +857,7 @@ export type K8sKind = {
   verbs?: K8sVerb[];
   shortNames?: string[];
   badge?: BadgeType;
+  color?: string;
```

Comment:
`color` is a bit vague. I asked Sam about it and he said "_maybe `iconColor`_" but he didn't seem 100% convinced.

I'm willing to let this in right now and we can brainstorm a better name. Some others that were thrown around with @andrewballantyne were: `themeColor`, `identityColor`...